pkg/chart/chart.go

```diff
@@ -70,7 +70,7 @@ func (c *chart) RenderTemplate(opts *TemplateOptions) (Template, error) {
 	if err != nil {
 		return nil, err
 	}
-	e := helmEngine.New(nil)
+	e := helmEngine.Engine{}
```

Comment:
I think I understand why this change was made, but since this is an unexpected use of `Engine`, a quick comment explaining why we need the `lookup` func in the template function map would be helpful to people who need to work on this in the future. Maybe something like:
```
// We do not use `helmEngine.New()` here because it sets Engine.clientProvider to a non-nil
// value. This causes the lookup function in the template function map to be set, which is
// undesirable in the case of hull, where there is no k8s cluster to look up resources on.
```

Reply:
added the comment in the file.